release.nix

```
@@ -14,6 +14,19 @@ let
   vuizvuiShortRev = vuizvuiSrc.shortRev or "abcdefg";
   vuizvuiVersion = "pre${toString vuizvuiRevCount}.${vuizvuiShortRev}";
 
+  # version of the nixos-unstable channel to get programs.sqlite from
+  programsSqliteVersion = "21.05pre269929.ff96a0fa563";
+  programsSqliteSha256 = "0ip26j2h11n1kgkz36rl4akv694yz65hr72q4kv4b3lxcbi65b3p";
+  programsSqlite = (import nixpkgs {}).fetchzip {
+    name = "programs.sqlite-${programsSqliteVersion}";
+    url = "https://releases.nixos.org/nixos/unstable/nixos-${programsSqliteVersion}/nixexprs.tar.xz";
+    sha256 = programsSqliteSha256;
+    downloadToTemp = true;
+    postFetch = ''
+      tar -xf $downloadedFile nixos-${programsSqliteVersion}/programs.sqlite > $out
```

Hm, this shouldn't work because it doesn't extract to stdout. Did you want to do something like this?

```suggestion
      tar -xOJf "$downloadedFile" nixos-${programsSqliteVersion}/programs.sqlite > "$out"
```

---

Derp 🤦🏻